src/Compiler/Compiler.h

```diff
@@ -92,6 +92,7 @@ class Compiler {
   enum class PowerOpt { None, Clean };
   enum class STAOpt { None, Clean, View, Opensta };
```

Comment:
please check if we really need STAOpt::Opensta. It looks like duplication.

Reply:
Will do, I was going to ask Alain and Tao to review the new option because currently STAOpt overloads clean and run operations which can result in the Opensta settings getting lost.